crates/holochain/tests/speed_tests/mod.rs

```diff
@@ -0,0 +1,330 @@
+//! # Speed tests
+//! These are designed to diagnose performance issues from a macro level.
+//! They are not intended to detect performance regressions or to be run in CI.
+//! For that a latency test or benchmark should be used.
+//! These tests are useful once you know there is an issue in locating which
+//! part of the codebase it is.
+//! An example of running the flame test to produce a flamegraph is:
+//! ```fish
```

Comment:
Could a `bash` example be provided in documentation if another shell is being used? It's easier for other shell users if the command is reasonably portable.

Reply:
Unfortunately I think the diff shows some new files that I didn't actually add...If anyone has any insight let me know but I'll have to look into this.